src/features/in-app-messenger/contextual/templates/styles/PreviewTitleBodyTwoButtonsModalTemplate.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../../../../styles/variables';
+
+$width: 660px;
+$supported-width: 1.1 * $width;
+
+.bdl-PreviewTitleBodyTwoButtonsModalTemplate {
+    @media only screen and (max-width: $supported-width - 1px) {
+        display: none;
+    }
+
+    letter-spacing: .3px;
+
+    .bdl-PreviewTitleBodyTwoButtonsModalTemplate-previewContainer {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 20px 0;
+
+        .MessagePreviewContent {
+            width: 400px;
+            max-height: 200px;
+
+            .MessagePreviewGhost-ghost {
+                width: 400px;
+                max-height: 200px;
+            }
+        }
+
+        .bp-container {
+            max-width: 400px;
+            max-height: 200px;
+
+            .bp {
+                background-color: $white;
+            }
+        }
+    }
+
+    .modal-actions {
```

Comment:
should these styles be more specific so that they don't apply to all `.modal-actions`? such as `.bdl-PreviewTitleBodyTwoButtonsModalTemplate-title .modal-actions`

Reply:
They are inside `.bdl-PreviewTitleBodyTwoButtonsModalTemplate` block, so it's like `.bdl-PreviewTitleBodyTwoButtonsModalTemplate .modal-actions`. The style should be applied only for modal with this specific template.

Reply:
got it, I didn't see it was wrapped, thanks!